### DISCUSSION TITLE
When adding multiple books via the "root audiobook folder" each books takes several minutes to show, but the user don't know it.

### DISCUSSION THREAD

**Discussion by WiliTest** [PaulWoitaschek/Voice#1505] [Ideas]
### What happened?

When I add multiple books via the "root audiobook folder" that contains 5 books, I expect to see all the books name appearing. But only a single one is showing. After 2-3 minutes, another appears then another, I'm still waiting for the others. 

I understand that things need to be processed, but there is no way to know that it's doing something (so I tried again, and again, until I figured out by chance that I need to wait). It would be better to display the name of each folder, and then to display a message "wait for it".  

### Version

6.1.0-cr2

**Comment by metbic** [comment]
Yes, a progress bar would be really nice. lacking it had led me to the deleted folder problem.
